packages/rum-core/src/domain/tracing/tracer.spec.ts

```diff
@@ -189,6 +189,48 @@ describe('tracer', () => {
       ])
     })
 
+    it('should preserve original headers contained in a Request instance', () => {
```

Comment:
it could be nice to have a test with a real fetch to ensure that this strategy works (cf previous implementation)